packages/core/src/card/card.ts

```diff
@@ -0,0 +1,9 @@
+export const cardPrefix = 'mzn-card';
+
+export const cardClasses = {
+  host: cardPrefix,
+  meta: `${cardPrefix}__meta`,
+  content: `${cardPrefix}__metaContent`,
+  header: `${cardPrefix}__metaContentHeader`,
+  actions: `${cardPrefix}__metaActions`,
```

Comment:
Since actions is Another component, it could have its own prefix and branches of classes.
ex: 
`export const cardActionPrefix = '${cardPrefix}-action'`
`action: cardActionPrefix`